js/files.js

```diff
@@ -271,7 +285,11 @@ async function updateVectorStore(vs_id, files) {
         { method: 'PUT', headers: {'Content-Type': 'application/json'}, body: JSON.stringify(request) }).
         then(r => {
             if (!r.ok) {
-                throw new Error ('Create Vector Store failed:' + r.statusText);
+                return r.json().then(e => {
+                    throw new Error(`${e.error}: ${e.message}`);
+                }).catch(() => {
+                    throw new Error ('Update Vector Store failed:' + r.statusText);
```

Comment:
unless this text is programmatically important
```suggestion
                    throw new Error ('Failed to create Vector Store:' + r.statusText);
```